src/test/java/pages/RoomPage.java

```diff
@@ -7,7 +7,7 @@ public class RoomPage {
     public static By calendarTable = By.xpath("//div[@class='calendar-table']");
     public static By invalidDate = By.xpath("(//div[@class='calendar-table']//td[contains(@class,'disabled')])[1]");
     public static By activeDate = By.xpath("//td[contains(@class,'active') or contains(@class,'available')]");
-    public static By roomView = By.xpath("//div[contains(@class,'maghny-gd')]/descendant::h3[@class='room-title']");
+    public static By roomView = By.xpath("//div[contains(@class,'maghny-gd')]");
```

Comment:
change heading, you can call this a room tile or container

Reply:
Done